src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const changedObj = {};
+
+  for (const key in robot) {
+    if (changedObj.hasOwnProperty(robot[key])) {
+      return null;
+    }
+    changedObj[robot[key]] = key;
```

Comment:
formatting + the checklist, p.1
```suggestion
    if (changedObj.hasOwnProperty(robot[key])) {
      return null;
    }

    changedObj[robot[key]] = key;
```